tr-tr/edn-tr.html.markdown

```diff
@@ -0,0 +1,156 @@
+---
+language: edn
+filename: learnedn.edn
+contributors:
+  - ["Seçkin KÜKRER", "https://github.com/LeaveNhA"]
+---
```

Comment:
Please add the contributors field from the original English doc here and add your name in the translators section. Additionally, please add the following lang line -
```
lang: tr-tr
```

Reply:
Firstly, this is an independent documentation written by me.
Nothing was used in it.
Still required to add original document's contributors ?

Reply:
And, this document is not a `Translation`, It's original btw.

Reply:
Fixed!

Reply:
> Firstly, this is an independent documentation written by me.
> Nothing was used in it.
> Still required to add original document's contributors ?

My apologies, you can remove the original document's contributors

Reply:
> And, this document is not a `Translation`, It's original btw.

Please put this somewhere in the PR next time.

Reply:
> Additionally, please add the following lang line -
> 
> ```
> lang: tr-tr
> ```

This change still needs to be made.

Reply:
> My apologies, you can remove the original document's contributors
No problem.

> This change still needs to be made.
It's done! 👨🏻‍💻